Automattic/wp-calypso
# WoA Logging i1: Add starter toolbar with date range picker

Closes https://github.com/Automattic/dotcom-forge/issues/1994

## Proposed Changes
This PR is a starter to push the project forward, we will refactor as more things are added to make it better. As it stands this PR:

* Adds date range picker to the toolbar.
* Fetches new logs based on the selected date range.

## Testing Instructions

- Go to `/site-logs/:atomicSite`
- Verify default date range is `now - 7days`
- Choose a date range and verify new logs are fetched for that range (_given your site has logs for that date_)

**Note** The logs are paginated so will not see all the logs for the selected dates.

![image](https://user-images.githubusercontent.com/47489215/227619663-58df139d-748f-4361-a975-7327e388fb10.png)


## Pre-merge Checklist
